product_variant_default_code/models/product.py

```diff
@@ -62,9 +62,9 @@ def render_default_code(product, mask):
 class ProductTemplate(models.Model):
     _inherit = 'product.template'
 
-    prefix_code = fields.Char(
-        string='Prefix code',
-        help='Add prefix to product variant default code',
+    code_prefix = fields.Char(
+        string='Reference Prefix',
+        help='Add prefix to product variant reference (default code)',
```

Comment:
Why renaming the field?

Reply:
https://github.com/OCA/product-variant/pull/75#discussion_r140209347

Reply:
OK, but then for preserving compatibility if migrating for old modules put `oldname='prefix_code'`.

Reply:
ok